kzg/src/lib.rs

```diff
@@ -12,4 +11,82 @@ pub type Uniq = blst::blst_uniq;
 pub type G1 = blst::blst_p1;
 pub type G2 = blst::blst_p2;
 
-pub mod finite;
+pub trait Fr: Clone {
+    fn default() -> Result<Self, String>;
```

Comment:
Padarykit kad `default()` grąžintų `Self` nes man dabar reikia visur rašyti `Fr::default().unwrap()` :D

Reply:
Šitą apskritai atverčiau, manau paskubėjau įpushint